rosbridge_client/README.md

```diff
@@ -0,0 +1,3 @@
+# Rosbridge Client
+
+It provides client mode of rosbridge which connects to the server as a client.
```

Comment:
Perhaps most of the contents of your comment https://github.com/RobotWebTools/rosbridge_suite/pull/360#issuecomment-431278180 can be moved in this README. The graph image helps being included here as well.

Reply:
which ROSBRDIGE version are u using.....?